### PR TITLE
Add FXIOS-9716 [Native Error Page] Strings for No internet Connection and Generic Error

### PR DESCRIPTION
**Pull request by tusharC95**
## :scroll: Tickets
[Jira ticket](https://mozilla-hub.atlassian.net/browse/FXIOS-9716)
[Github issue](https://github.com/mozilla-mobile/firefox-ios/issues/21354)

## :bulb: Description
Added strings for no internet connection and generic error page.

## :pencil: Checklist
You have to check all boxes before merging
- [x] Filled in the above information (tickets numbers and description of your work)
- [x] Updated the PR name to follow our [PR naming guidelines](https://github.com/mozilla-mobile/firefox-ios/wiki/Pull-Request-Naming-Guide)
- [ ] Wrote unit tests and/or ensured the tests suite is passing
- [ ] When working on UI, I checked and implemented accessibility (minimum Dynamic Text and VoiceOver)
- [ ] If needed, I updated documentation / comments for complex code and public methods
- [ ] If needed, added a backport comment (example `@Mergifyio backport release/v120`)

### CODE REVIEW

File: firefox-ios/Client/Frontend/Strings.swift [hunk 1]
```diff
@@ -1466,6 +1466,34 @@ extension String {
     }
 }
 
+// MARK: - Native Error Page
+extension String {
+    struct NativeErrorPage {
+        struct GenericError {
```
Comment (cyndichin) [inline firefox-ios/Client/Frontend/Strings.swift:1472]:
```suggestion
        struct NoInternetConnection {
```

File: firefox-ios/Client/Frontend/Strings.swift [hunk 2]
```diff
@@ -1466,6 +1466,34 @@ extension String {
...
+            public static let NoInternetConnectionTitleLabel = MZLocalizedString(
+                key: "NativeErrorPage.GenericError.NoInternetConnection.TitleLabel.v136",
+                tableName: "NativeErrorPage",
+                value: "Looks like there’s a problem with your internet connection.",
+                comment: "On error page, this is the title for no internet connection")
+            public static let NoInternetConnectionDescription = MZLocalizedString(
+                key: "NativeErrorPage.GenericError.NoInternetConnection.Description.v136",
```
Comment (cyndichin) [inline firefox-ios/Client/Frontend/Strings.swift:1479]:
```suggestion
                key: "NativeErrorPage.NoInternetConnection.Description.v131",
```

File: firefox-ios/Client/Frontend/Strings.swift [hunk 3]
```diff
@@ -1466,6 +1466,34 @@ extension String {
...
+                tableName: "NativeErrorPage",
+                value: "Try connecting on a different device. Check your modem or router. Disconnect and reconnect to Wi-Fi.",
+                comment: "On error page, this is the description for no internet connection.")
+            public static let NoErrorCodeTitleLabel = MZLocalizedString(
+                key: "NativeErrorPage.GenericError.NoErrorCode.TitleLabel.v136",
```
Comment (cyndichin) [inline firefox-ios/Client/Frontend/Strings.swift:1484]:
```suggestion
                key: "NativeErrorPage.GenericError.NoErrorCode.TitleLabel.v131",
```

File: firefox-ios/Client/Frontend/Strings.swift [hunk 4]
```diff
@@ -1466,6 +1466,34 @@ extension String {
...
+                tableName: "NativeErrorPage",
+                value: "Be careful. Something doesn’t look right.",
+                comment: "On error page, this is the title for generic error with no error code.")
+            public static let PageReload = MZLocalizedString(
+                key: "NativeErrorPage.GenericError.PageReload.v136",
```
Comment (cyndichin) [inline firefox-ios/Client/Frontend/Strings.swift:1489]:
```suggestion
                key: "NativeErrorPage.ButtonLabel.v131",
```

File: firefox-ios/Client/Frontend/Strings.swift [hunk 5]
```diff
@@ -1466,6 +1466,34 @@ extension String {
...
+                tableName: "NativeErrorPage",
+                value: "Reload",
+                comment: "On error page, this is shown on a button that will try to load the page again.")
```
Comment (cyndichin) [inline firefox-ios/Client/Frontend/Strings.swift:1492]:
```suggestion
                comment: "On error page, this is the text on a button that will try to load the page again.")
```

**Review by cyndichin — COMMENTED** [review]
thanks for the string changes! left a couple of comments. Maybe we should separate based on 
- internet connection
- generic error
and then the button label can live on the native error page level. Let me know what you think!

File: firefox-ios/Client/Frontend/Strings.swift (hunk 5 above)
Comment (tusharC95) [reply firefox-ios/Client/Frontend/Strings.swift:1492]: Thanks, I will update the text as suggested.

File: firefox-ios/Client/Frontend/Strings.swift [hunk 6]
```diff
@@ -1466,6 +1466,41 @@ extension String {
...
+                key: "NativeErrorPage.GenericError.Description.v131",
+                tableName: "NativeErrorPage",
+                value: "An SSL error has occurred and a secure connection to the server cannot be made.",
+                comment: "On error page, this is the description for generic error.")
+        }
+        public static let ButtonLabel = MZLocalizedString(
+            key: "NativeErrorPage.ButtonLabel.v131",
+            tableName: "NativeErrorPage",
+            value: "Reload",
+            comment: "On error page, this is the text on a button that will try to load the page again.")
```
Comment (cyndichin) [inline firefox-ios/Client/Frontend/Strings.swift:1500]: nit: maybe move to the top level so its not lost within the struct

**Review by cyndichin — APPROVED** [review]
thanks for making the changes, looks great!

**Pull request merged**